smrf/envphys/solar/toporad.py

```diff
@@ -0,0 +1,285 @@
+import numpy as np
+from topocalc.horizon import horizon
+from topocalc.shade import shade
+
+from smrf.envphys.solar.twostream import twostream
+from smrf.envphys.solar.irradiance import direct_solar_irradiance
+from smrf.envphys.thermal.topotherm import hysat
+from smrf.envphys.albedo import albedo
+from smrf.envphys.constants import SEA_LEVEL, STD_LAPSE, \
+    GRAVITY, MOL_AIR, STD_AIRTMP
+
+VISIBLE_MIN = .28
+VISIBLE_MAX = .7
+IR_MIN = .7
+IR_MAX = 2.8
+
```

Comment:
Same constants have been defined in `smrf/distribute/solar.py`
Combine?

Reply:
Or move to `constants.py`?

Reply:
Moved to constants.